### PR TITLE
sim: launch_openpilot.sh fix and YUV to NV12 output

### PR DESCRIPTION
**Pull request by jeroenlammersma**
- Fix for openpilot not launching during simulation: #25493 

- Output images in NV12 format during simulation: #25451 

Converting is done from  YUV to NV12, not from RGB to NV12 directly.
I tried changing [rgb_to_yuv.cl](https://github.com/commaai/openpilot/blob/master/system/camerad/transforms/rgb_to_yuv.cl) to [real_debayer.cl](https://github.com/commaai/openpilot/blob/master/system/camerad/cameras/real_debayer.cll), but I am not able to compile the cl program. Also I am not sure if that will work.

Nevertheless, a todo is
- swapping to real_debayer.cl or creating a dedicated rgb_to_nv12.cl program.

This is also something I tried to create, but I had the same issue as mentioned above (see image).

clBuildProgram failed

I know what's going wrong, however I don't know the proper way to fix this in the codebase (I thought SCons would handle building).

### CODE REVIEW

File: selfdrive/loggerd/bootlog.cc [hunk 1]
```diff
@@ -24,20 +25,22 @@ static kj::Array<capnp::word> build_boot_log() {
     i++;
   }
 
-  // Gather output of commands
-  std::vector<std::string> bootlog_commands = {
-    "[ -e /dev/nvme0 ] && sudo nvme smart-log --output-format=json /dev/nvme0",
-    "[ -x \"$(command -v journalctl)\" ] && journalctl",
-  };
+  if (!SIMULATION) {
```
Comment (adeebshihadeh) [inline selfdrive/loggerd/bootlog.cc:28]: why is this hanging? I don't see why this is simulator specific; it doesn't hang on my machine

File: tools/sim/bridge.py [hunk 2]
```diff
@@ -101,15 +101,22 @@ def _cam_callback(self, image, frame_id, pub_type, yuv_type):
...
     # convert RGB frame to YUV
     rgb = np.reshape(img, (H, W * 3))
     rgb_cl = cl_array.to_device(self.queue, rgb)
     yuv_cl = cl_array.empty_like(rgb_cl)
     self.krnl(self.queue, (np.int32(self.Wdiv4), np.int32(self.Hdiv4)), None, rgb_cl.data, yuv_cl.data).wait()
     yuv = np.resize(yuv_cl.get(), rgb.size // 2)
-    eof = int(frame_id * 0.05 * 1e9)
 
-    self.vipc_server.send(yuv_type, yuv.data.tobytes(), frame_id, eof, eof)
+    # convert YUV to NV12
```
Comment (adeebshihadeh) [inline tools/sim/bridge.py:112]: can you split this out into a separate PR?

Comment (jeroenlammersma) [reply tools/sim/bridge.py:112]: Will do.

File: selfdrive/loggerd/bootlog.cc (hunk 1 above)
Comment (jeroenlammersma) [reply selfdrive/loggerd/bootlog.cc:28]: I will do some more experimenting, if the issue persists, I will come back to this later.

**Comment by jeroenlammersma** [comment]
updated requirements and now it works again. Case closed.
